fetch.bs

```diff
@@ -4082,10 +4082,6 @@ Entries may be removed before that moment arrives.
  <li>
   <p>If <var>credentials</var> is `<code>true</code>`, return success.
```

Comment:
This `<li>\n  <p>` should become `<li><p>` since the `<li>` no longer contains multiple `<p>`s.